src/main/features/claim/views/eligibility/single-claimant.njk

```diff
@@ -0,0 +1,40 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, errorSummary, radioGroup, saveAndContinueButton %}
+
+{% set title = t('Single Claimant | Money claim') %}
+
+{% block content %}
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-large">{{ t('Are you the only person or business making the claim?') }}</h1>
+      <p>{{ t('You can’t use this service if more than one person or business is making the claim') }}</p>
+
+      {% set name = 'singleClaimant' %}
+
+      <form novalidate method="post" class="analytics-submit-event-trigger"
+            data-event-action="Eligibility: Select Yes"
+            data-event-label-from="{{ name }}">
+        {{ csrfProtection(csrf) }}
+
+        {{
+          radioGroup(
+            'Are you the only person or business making the claim?',
```

Comment:
Please implicitly name argument names for readability e.g.:

```
radioGroup(
  label = 'Are you the only person or business making the claim?',
  ...
```

Reply:
I was working off the Over_18 file which didn't have that. 
I will add it in now